maskit/examples/load_data.py

```diff
@@ -0,0 +1,22 @@
+from maskit.examples.circles import circles
+from maskit.examples.iris import iris
+from maskit.examples.mnist import mnist
+
+
+def load_data(dataset, **kwargs):
```

Comment:
As this serves as the main entry function to load a specific dataset, it would be great to have some type annotations and a short doc string.

Reply:
A general comment, please consider moving this function into `__init__.py` within the submodule. You might even consider renaming the submodule from `examples` to `datasets` (as for example done in sklearn).